_posts/2017-06-09-the-lead-developer-conference.md

```diff
@@ -260,6 +260,44 @@ Katherine brought an interesting and new perspective of (mis)communication betwe
+
+Anjuan compares this with his decision to become a software developer and the trials he had to face and the experience he had to gain. He received certain gifts like structured, logic thinking that he uses to be a better leader.
+
+The part that drew him most to Campbell's model is the hero–mentor part. He realised the mentor has completed the hero's story and is now helping someone else do it. In a way, the mentor is passing the baton to the hero.
+
+Actual baton's don't play a big role in real life, but Anjuan has found that something that comes close is **influence**. If he can improve this skill, he feels he will become a better leader.
+
+#### The law of influence
+
+Anjuan refers to the book titled The 21 irrefutable laws of leadership, by John C. Maxwell. The key point in this book is the law of influence:
+    
+> The true measure of leadership is influence. Nothing more. Nothing less.
+
+You can be a leader that wields his title, or a leader that gets buy-in of the team by using influence.
+    
+James MacGregor Burns describes it in his book Leadership as _transactional_ (carrot and stick) vs. _transformational_ (charisma to influence) leadership.
+
+How does one apply influence in his role as a leader? Anjuan takes inspiration in the values of the Agile manifesto:
+
+- _Individuals and interactions over processes and tools._ People build software, processes and tools don't. How do you get and keep a team motivated? Anjuan has found that "Preserve diginity at all costs" builds team influence. You don't have to all agree, but you have to respect everyone in your team.
+
+- _Working software over comprehensive documentation._ "Working always ships faster than perfect" means that you have to start with something, and improve it along the way. Focussing on getting things working, gains build influence. This applies to anything in your role as a leader.
+    
+- _Customer collaboration over contract negotiation._ Customers trust colleagues, not contracts, so treat your customers as colleagues. Nobody reads contracts, don't expect people to read them. Communicate with people directly. But not all your customers are your friends. Cut toxic customers.
+
+- _Responding to change, instead of following a plan._ We can't predict the future, so don't make plans to far ahead. "Don't fear surprises, fear inflexibility." Iterate and improve. By doing this he improved his scheduling influence, because he delivered sooner.
```

Comment:
typo: _to far ahead_ -> _too far ahead_

I don't understand this sentence: _By doing this he improved his scheduling influence, because he delivered sooner._